test/src/e2e.long/mempool.test.ts

```diff
@@ -38,7 +38,10 @@ describe("Memory pool size test", function() {
             sending.push(nodeA.sendPayTx({ seq }));
         }
         await Promise.all(sending);
-        const pendingTransactions = await nodeA.sdk.rpc.chain.getPendingTransactions();
+        const pendingTransactions = await nodeA.sdk.rpc.sendRpcRequest(
+            "mempool_getPendingTransactions",
+            [null, null, true]
```

Comment:
Before this patch, the test would send `[null, null, false]`. I guess this was the cause of test failure.